### ISSUE TITLE
Kubernetes Dashboard v7+

### ISSUE THREAD

**Issue by davisonja** [OpenUnison/openunison-k8s#111]
The dashboard version took a big jump recentlyish, when 3.0.0-alpha turned into a [7.0.0 release](https://github.com/kubernetes/dashboard/releases/tag/kubernetes-dashboard-7.0.0) which involves a number of changes and means the Login Portal install instructions (obviously substituting the newer version in place of the 2.7 install) don't work.

Are there plans to add instructions/support for the newer 7+ versions?

**Comment by mlbiam** [comment]
> Are there plans to add instructions/support for the newer 7+ versions?

 Yes.  7.0 made a major change from having a single container and URL endpoint to two containers and wanting to use an Ingress, even internally.  Right now I'm going back and forth between using the existing helm chart, or just integrating it directly into the OpenUnison charts.

**Comment by mlbiam** [comment]
Just got the initial attempt working.  It's plain-text from OpenUnison --> the dashboard pods right now.  Need to get TLS integrated, get instructions created and documented.  But look for it early next weekish....

**Comment by mlbiam** [comment]
And done - https://openunison.github.io/deployauth/#the-kubernetes-dashboard

**Comment by kastl-ars** [comment]
> And done - https://openunison.github.io/deployauth/#the-kubernetes-dashboard

Thanks a lot, Marc!

**Comment by kastl-ars** [comment]
Hmmm, this does not seem to work for me. Using the dashboard helm chart and the [values.yaml provided in your link](https://openunison.github.io/assets/yaml/kubernetes-dashboard-values.yaml) I end up with OpenUnison trying to reach a service called `kubernetes-dashboard.kubernetes-dashboard.svc` (as noted in the orchestra pod logs).

Which in my case does not exist:

```
$ kubectl get svc -n kubernetes-dashboard
NAME                                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
kubernetes-dashboard-api               ClusterIP   10.43.145.215   <none>        8001/TCP   66m
kubernetes-dashboard-auth              ClusterIP   10.43.103.136   <none>        8000/TCP   66m
kubernetes-dashboard-metrics-scraper   ClusterIP   10.43.96.110    <none>        8000/TCP   66m
kubernetes-dashboard-web               ClusterIP   10.43.115.229   <none>        8001/TCP   66m
```

I tried setting the `dashboard.service_name` in the OpenUnison `values.yaml` to each of the existing services (api, auth, web), but then the dashboard does not open with an error but loads indefinitely.

I am using the `https://kubernetes.github.io/dashboard/` helm repository and installing the `kubernetes-dashboard` chart like mentioned in your documentation link. Seems like version 7.5.0 is the current installed one.

Kind Regards,
Johannes

P.S.: There is a small typo in the documentation: `so there's some specific customizations to the dashboard's helm values file that needs to be made` should read `that need to be made` (`need` without an `s`).

**Comment by mlbiam** [comment]
@kastl-ars does your openunison values.yaml include `dashbaord.new: true`?

**Comment by kastl-ars** [comment]
OK, adding `dashboard.new: true` to the values.yaml seems to do the trick.

Could you please make that more prominent in the documentation? Maybe add a stanza in the **The Kubernetes Dashboard** section that this parameter is required if you install the dashboard via Helm chart (and not with the old AIO manifest)?

Have a nice weekend, Marc!
